tasks.rb

```diff
@@ -229,6 +229,11 @@ def apply_hpxml_modification(hpxml_file, hpxml)
   hpxml.header.xml_generated_by = 'tasks.rb'
   hpxml.header.created_date_and_time = Time.new(2000, 1, 1, 0, 0, 0, '-07:00').strftime('%Y-%m-%dT%H:%M:%S%:z') # Hard-code to prevent diffs
 
+  # Logic that can only be applied based on file name
+  if ['base-schedules-detailed-occupancy-stochastic-hpxml-header.xml'].include? hpxml_file
+    hpxml.header.schedules_filepaths << '../../HPXMLtoOpenStudio/resources/schedule_files/occupancy-stochastic.csv'
+  end
```

Comment:
Demonstrate backward compatibility.

Reply:
If you make the change I proposed where the BuildResSchedule measure sometimes writes to the HPXML Header and sometimes to the Building Header, I assume we can remove this new HPXML sample file as we'd end up testing both pathways through our existing sample files.

Reply:
Also, it feels like we're missing a sample file where we actually vary the schedules across the HPXML buildings. How about creating a new `base-multiple-buildings-detailed-occupancy-stochastic.xml` based on `base-multiple-buildings.xml`?

Reply:
Hmm, I'm realizing that the problem with your first comment here is that tasks.rb uses `BuildResidentialHPXML` to first create the HPXML (with SchedulesFilePath at building header) and _then_ uses `BuildResidentialScheduleFile` to re-write the HPXML file (with SchedulesFilePath at the HPXML header)...

Reply:
I suppose I can make a similar change to `BuildResidentialHPXML`?

Reply:
Ugh, even then it gets complicated. You'd have to "move" a schedule from HPXML header to building header when going from 1 building to 2 buildings using `BuildResidentialHPXML`...

Reply:
When we create sample files, there's no need to specify a schedule file path to BuildResidentialHPXML if we are going to use BuildResidentialScheduleFile to create the schedules (and add the path), no? Or maybe I'm not following.

Reply:
But tasks.rb uses the `schedules_filepaths` argument to queue up running `BuildResidentialScheduleFile`. Even so, (it's not in this PR) but say you had an HPXML file with a single building (and SchedulesFilePath is in the HPXML header). If you went to   add a building to the HPXML file, then you're assuming the existing schedule file applies to the second building?

Reply:
I'd propose we save addressing this in https://github.com/NREL/OpenStudio-HPXML/pull/1457.